shell/synchronize/etc/ditf.cfg

```
@@ -1,6 +1,6 @@
-source_dir='/groups/solve-rd/prm10/sftp/ditf'
-destination_dir='/groups/umcg-solve-rd/scr01/'
-destination_machine='umcg-${ATEAMBOTUSER}@cher-ami.hpc.rug.nl'
+ksource_dir='/groups/solve-rd/prm10/sftp/ditf'
```

Is this correct, or should it be
```suggestion
source_dir='/groups/solve-rd/prm10/sftp/ditf'
```
?